package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "eslint-config-bravissimo",
-  "version": "2.0.0",
+  "version": "2.0.1",
```

Comment:
Are there any backwards compatibility issues with the airbnb upgrade? I suspect this should be 2.1.0 or maybe even 3.0.0 - it's not a minor patch either way.